ui/AssessmentManager.py

```diff
@@ -38,24 +37,23 @@ def finished_init(self, *args):
 
 
     def _load(self, module_number: int):
-        filepath = "assets/json/questions" + str(module_number) + ".json"
+        filepath = "assets/json/questions0.json"
```

Comment:
should this always be questions0 or should we keep this as it was?

Reply:
ill keep it as is